docker-compose.prod-run.yml

```diff
@@ -3,9 +3,21 @@ version: "3.7"
 services:
   frontend:
     image: etimodanielwinther/diamonds2_frontend:${DIAMONDS_DOCKER_TAG}
-    command: /bin/sh -c "nginx -g 'daemon off;'"
+    command: '/bin/sh -c ''while :; do sleep 6h & wait $${!}; nginx -s reload; done & nginx -g "daemon off;"'''
```

Comment:
Perhaps use a multiline command in docker-compose instead. Makes it easier to read and the future diffs will be smaller as well.

Reply:
Changed to multiline